data-reconciliation-app/test/test.sh

```diff
@@ -14,6 +15,7 @@ function usage {
     echo "  --nodeAddress        string      The IP and port of the primary CCF node"
     echo "  --certificate_dir    string      The directory where the certificates are"
     echo "  --interactive        boolean     Optional. Run in Demo mode"
+    echo "  --ts_mode            boolean     Optional. Run in Typescript mode"
```

Comment:
Because it's a boolean - I would prefer `--typescript` as it reads better - but I appreciate that this could become the default!

Reply:
done